src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -1952,18 +1953,33 @@ void c_typecheck_baset::typecheck_side_effect_function_call(
         auto gcc_polymorphic = typecheck_gcc_polymorphic_builtin(
           identifier, expr.arguments(), f_op.source_location()))
       {
-        if(!symbol_table.has_symbol(gcc_polymorphic->get_identifier()))
+        irep_idt identifier_w_type = gcc_polymorphic->get_identifier();
```

Comment:
Expand `_w_` here and below? Is that `_write_` or something else?

Reply:
It's "with" - I'll spell that out :-)